airbyte-cdk/java/airbyte-cdk/core/src/testFixtures/java/io/airbyte/cdk/integrations/base/ssh/SshBastionContainer.java

```diff
@@ -4,8 +4,6 @@
 
 package io.airbyte.cdk.integrations.base.ssh;
 
-import static io.airbyte.cdk.integrations.base.ssh.SshHelpers.getInnerContainerAddress;
-import static io.airbyte.cdk.integrations.base.ssh.SshHelpers.getOuterContainerAddress;
```

Comment:
These methods, which take testcontainer args, don't belong in `SshHelpers` because that class is not a test fixture.